src/iop/temperature.c

```diff
@@ -862,8 +862,8 @@ static void _color_finetuning_slider(dt_iop_module_t *self)
         max_tune[ch] = neutral[ch] * wb_max->channels[ch];
       }
 
-      const float maxsRGBmin_tune = fmaxf(fmaxf(min_tune[0], min_tune[1]), min_tune[2]);
-      const float maxsRGBmax_tune = fmaxf(fmaxf(max_tune[0], max_tune[1]), max_tune[2]);
+      const float maxsRGBmin_tune = max3(min_tune);
```

Comment:
So the fminf was wrong as the array is of doubles?

Reply:
That would avoid two conversions, so it will save a bit of time for each pixel.